### ISSUE TITLE
Link checker failing on Wheeler_Kiladis

### ISSUE THREAD

**Issue by tsjackson-noaa** [NOAA-GFDL/MDTF-diagnostics#74]
**Issue:** Framework reports missing files in the Wheeler_Kiladis POD's output (filenames linked to from HTML page are not present in POD's output).

Issue was reported by @jkrasting -- I haven't been able to reproduce this using the CESM sample data (QBOi.EXP1.AMIP.001) and either the current /main or /develop. This is on mac os and not using a pristine install of the framework, so I might be missing something.

John, if you've got the logs from that run handy, could you post them here? Another thing to check would be if the plots really are getting saved to the Wheeler_Kiladis output directory, but with different filenames than what's given in the HTML links.

**Comment by jkrasting** [comment]
@tsjackson-noaa - I think it might be link checker.  I see these messages but the files appear to be present:

```
Failed to find file or connect to server.
Reason:  [Errno 2] No such file or directory: '/Users/krasting/mdtf/wkdir/MDTF_QBOi.EXP1.AMIP.v2.001_1977_1981/Wheeler_Kiladis/model/Fig1.Asym.{CASENAME}_rlut.png'

Failed to find file or connect to server.
Reason:  [Errno 2] No such file or directory: '/Users/krasting/mdtf/wkdir/MDTF_QBOi.EXP1.AMIP.v2.001_1977_1981/Wheeler_Kiladis/model/Fig1.Sym.{CASENAME}_rlut.png'

Failed to find file or connect to server.
Reason:  [Errno 2] No such file or directory: '/Users/krasting/mdtf/wkdir/MDTF_QBOi.EXP1.AMIP.v2.001_1977_1981/Wheeler_Kiladis/model/Fig2.{CASENAME}_rlut.png'

Failed to find file or connect to server.
Reason:  [Errno 2] No such file or directory: '/Users/krasting/mdtf/wkdir/MDTF_QBOi.EXP1.AMIP.v2.001_1977_1981/Wheeler_Kiladis/model/Fig3.Asym.{CASENAME}_rlut.png'

Failed to find file or connect to server.
Reason:  [Errno 2] No such file or directory: '/Users/krasting/mdtf/wkdir/MDTF_QBOi.EXP1.AMIP.v2.001_1977_1981/Wheeler_Kiladis/model/Fig3.Sym.{CASENAME}_rlut.png'
```

There are other legitimate errors when running with the 15ºx15º data.  See attached log.

[Wheeler_Kiladis.log.txt](https://github.com/NOAA-GFDL/MDTF-diagnostics/files/5596694/Wheeler_Kiladis.log.txt)

**Comment by tsjackson-noaa** [comment]
Closing because this was fixed by PR NOAA-GFDL/MDTF-diagnostics#46. Just verified that there are no missing files when running Wheeler_Kiladis from current `/develop` on sample model data (with caveat that obs data must also be updated to the current version from ftp.cgd.ucar.edu.)
